tsconfig.json

```diff
@@ -8,24 +9,21 @@
         "strictFunctionTypes": true,
         "strictPropertyInitialization": true,
         "noImplicitThis": true,
-        "noImplicitAny": true,
+        "noImplicitAny": false,
```

Comment:
Let's agree and bring this back after web3 package update (I am assuming this should be the only reason).